### PR TITLE
UPSAddBattTime2

### PR DESCRIPTION
**Pull request by briancjohns**

### CODE REVIEW

**Review by cstirdivant — DISMISSED** [review]

**Comment by cstirdivant** [comment]
@briancjohns  please address validator error

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -2918,6 +2918,9 @@ literals:
   - LOW
   - MEDIUM
   - HIGH
+- remaining_battery_discharge_time_sensor:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:2921]: See validator message, this field name violates the ordering rules

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Review by briancjohns — COMMENTED** [review]
resolved the telemetry fields validation error

**Review by cstirdivant — APPROVED** [review]

**Comment by cstirdivant** [comment]
Scheduled merge for 3/28

**Pull request merged**